sockets/1.2/modules/ROOT/pages/sockets-connector-studio.adoc

```diff
@@ -0,0 +1,253 @@
+
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:sockets-documentation.adoc[Sockets Connector Reference].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Sockets Connector to your Mule project to automatically populate the XML code with the connector's namespace and schema location and add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type `Sockets` in the search field.
+. Click *Sockets* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+
+[[configure-input-source]]
+== Configure Socket Listener Source
+
+An input source initiates a flow when a specified condition is met. The Socket Listener operation represents a socket endpoint that clients can connect to. By browsing to this endpoint, external clients establish a TCP/UDP Socket connection and trigger the flow each time they send a message.
+
+. In the *Mule Palette* view, select *Socket* > *Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Click the plus sign (+) next to the *Connector configuration* field to configure a global element that can be used by all instances of `Socket Listener` in the app.
+. On the *General* tab, for *Connection* select `Tcp listener` or `Udp listener` connections.
+. Configure the specific tabs for the connection previously selected:
++
+[%header,cols=2]
+|===
+| Tab Name | Description
+| *General* | Set *Host* and *Port* required attributes, and configure any other optional fields.
+| *Buffer Configuration* | Optionally specify send and receive buffer size information.
+| *TLS* | Optionally specify TLS information.
+| *Timeout Configuration* | Optionally specify client and server timeout information.
+| *Advanced* | Optionally specify reconnection information, including a reconnection strategy.
+|===
+
+[start=7]
+. Click *Test Connection* to confirm that Mule can connect with the specific server.
+. Click *OK*.
+
+In Studio, the Socket Listener operation in the flow looks as follows:
+
+.Socket Listener Source
+image::socket-listener-source.png[Socket Listener Source]
+
+In Studio, configuring the Socket Listener with TCP connection to accept local incoming connections on port `8082`, looks as follows:
```

Comment:
Remove the comma after 8082'. 

I think it would be better to say:
"The following screenshot shows how to...
or
"The following screenshot shows an example of configuring..."

Global where this wording exists.